src/Infrastructure/Repositories/LocalFilesRepository.php

```diff
@@ -20,14 +20,15 @@ final class LocalFilesRepository implements FilesRepository
     /**
      * LocalFilesRepository constructor.
      *
-     * @param  \Symfony\Component\Finder\Finder  $finder
+     * @param \Symfony\Component\Finder\Finder  $finder
      */
     public function __construct(Finder $finder)
     {
         $this->finder = $finder
             ->files()
-            ->name(['*.php'])
-            ->exclude(['vendor', 'tests', 'Tests', 'test', 'Test'])
+            ->name(['*.php',])
+            ->exclude(['vendor', 'tests', 'Tests', 'test', 'Test',])
+            ->notName(['*.blade.php',])
```

Comment:
This should't go the laravel default preset?

Reply:
The reason for adding it in here is that some users are using blade outside of Laravel. For example @liepumartins mentioned it here https://github.com/nunomaduro/phpinsights/issues/59#issuecomment-493867231.